### PR TITLE
Add import requests

### PR DESCRIPTION
**Pull request by meowyen**
Add import requests

### CODE REVIEW

**Comment by bcizek91** [comment]
Test123

File: data_collection.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import requests
+
+print "This is an awesome test"
```
Comment (bcizek91) [inline data_collection.py:3]: Test123

**Review by bcizek91 — CHANGES_REQUESTED** [review]

File: data_collection.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import requests
```
Comment (tonyhartpdx) [inline data_collection.py:1]: Hello! Thanks!!

**Review by tonyhartpdx — APPROVED** [review]

File: data_collection.py (hunk 2 above)
Comment (meowyen) [reply data_collection.py:1]: You're welcome.

**Review by bcizek91 — APPROVED** [review]

**Pull request merged**